test/ui/widgets/grade_evaluation_tile_test.dart

```diff
@@ -9,8 +9,8 @@ import 'package:notredame/ui/widgets/grade_evaluation_tile.dart';
 import 'package:notredame/ui/widgets/grade_circular_progress.dart';
 
 // MODELS
-import 'package:notredame/core/models/evaluation.dart' as model;
-import 'package:notredame/core/models/course_summary.dart';
+import 'package:ets_api_clients/models.dart';
+import 'package:ets_api_clients/models.dart' as models;
```

Comment:
It's the second time you do that, is there a reason to import it two times??

```suggestion
import 'package:ets_api_clients/models.dart' as models;
```